packages/cli/src/commands/init/init.ts

```diff
@@ -98,6 +130,79 @@ async function createFromTemplate({
   );
 
   try {
+    await PackageManager.init({
+      preferYarn: !npm,
+      silent: true,
+      root: templateSourceDir,
+    });
+
+    const yarnVersion = getYarnVersionIfAvailable();
+
+    if (!npm && yarnVersion !== null) {
+      if (semver.satisfies(yarnVersion, '>=3.0.0')) {
+        const yarnrcYmlPathLocation = await findUp('.yarnrc.yml', {
+          type: 'file',
+        });
+
+        if (yarnrcYmlPathLocation && fs.existsSync(yarnrcYmlPathLocation)) {
+          const yarnYmlContent = parse(
+            fs.readFileSync(yarnrcYmlPathLocation, 'utf8'),
+          );
+
+          if (
+            !yarnYmlContent.nodeLinker ||
+            yarnYmlContent.nodeLinker !== 'node-modules'
+          ) {
+            await overwriteYarnrcFile(yarnrcYmlPathLocation, yarnYmlContent);
+          }
+
+          addNodeLinker(templateSourceDir);
```

Comment:
Won't this just overwrite the content in `overwriteYarnrcFile`?

Reply:
Nope, because to this function I'm passing template directory, `overwriteYarnrcFile` is changing content inside `.yarnrc.yml` file which lives in project.